lang/lua/lua.py

```diff
@@ -0,0 +1,312 @@
+from talon import Context, Module, actions, settings
+
+mod = Module()
+ctx = Context()
+ctx.matches = r"""
+tag: user.lua
+"""
+
+mod.setting(
+    "lua_version",
+    type=float,
+    default=5.1,
+    desc="The default lua version to use. Dictates certain operators",
+)
+mod.tag("stylua", desc="Tag for stylua linting commands")
+
+ctx.lists["user.code_common_function"] = {
+    "to number": "tonumber",
+    "eye pairs": "ipairs",
+    "print": "print",
+    "print F": "printf",
+    "type": "type",
+    "assert": "assert",
+    "get meta table": "getmetatable",
+    "set meta table": "setmetatable",
+    # io
+    "eye oh write": "io.write",
+    "eye oh read": "io.read",
+    "eye oh open": "io.open",
+    # string
+    "format": "string.format",
+    "string G find": "string.gfind",
+    "string find": "string.strfind",
+    "string len": "string.strlen",
+    "string upper": "string.strupper",
+    "string lower": "string.strlower",
+    "string sub": "string.strsub",
+    "string G sub": "string.gsub",
+    "string match": "string.match",
+    "string G match": "string.gmatch",
+    # table
+    "table unpack": "table.unpack",
+    "table insert": "table.insert",
+    "tabel get N": "table.getn",
+    "tabel sort": "table.sort",
+    # math
+    "math max": "math.max",
+    # json
+    "jason parse": "json.parse",
+    # http
+    "web get": "http.get",
```

Comment:
might be more consistent to use "http get" here, or at least have it as an alternative